core/tests/performance_framework/process_data.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script to process HAR data and timing data"""
+
+import datetime
+import dateutil.relativedelta
+
+
+class ProcessData(object):
+    """Class that contains methods to process data and get us metrics"""
+
+    def __init__(self, har=None, timings=None):
+        self.har = har
```

Comment:
After looking through the entire PR, I think I understand what this is. It seems to be a dict containing the same data a HAR file contains. It's not actually a HAR file, though. In fact, it's really a dict containing stats about a particular page loading session. Perhaps your code in this PR would be much clearer if you refer to it as page session stats rather than a 'har' (the HAR file just seems to contain the stats which populates this dict; in many ways the HAR file isn't very relevant in your implementation because it's a level of abstraction below this code).